### PR TITLE
Update slncky to v1.0.4

### PR DESCRIPTION
**Pull request by corburn**
* [x] I have read the [guidelines for bioconda recipes](https://bioconda.github.io/guidelines.html).
* [ ] This PR adds a new recipe.
* [x] AFAIK, this recipe **is directly relevant to the biological sciences** (otherwise, please submit to the more general purpose [conda-forge channel](https://conda-forge.org/docs/)).
* [x] This PR updates an existing recipe.
* [ ] This PR does something else (explain below).

### CODE REVIEW

File: recipes/slncky/meta.yaml [hunk 1]
```diff
@@ -1,34 +1,38 @@
...
+    - lastz
 
 test:
-    commands:
-        - slncky --help > /dev/null 2>&1
+  commands:
+    - slncky --help > /dev/null 2>&1
         # The annotations directory needs to be downloaded by the user.
         # It is too big to be downloaded automatically.
         # - test -d $PREFIX/bin/annotations
```
Comment (jerowe) [inline recipes/slncky/meta.yaml]: Add a post-link script that will install the annotations?

**Comment by bgruening** [comment]
A rebase against latest master can help.

**Comment by corburn** [comment]
@bgruening  rebased against master

**Comment by bgruening** [comment]
`FileNotFoundError: [Errno 2] No such file or directory: '/opt/conda/conda-bld/slncky_1529479011549/_h_env_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placeh/bin/browserHeader.txt'`

**Pull request merged**